<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.16">
<meta name="author" content="Francesco Ballesio Ph.D.">
<link rel="icon" type="image/x-icon" href="favicon.ico">
<title>Francesco Ballesio</title>
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700">
<style>
/*! Asciidoctor default stylesheet | MIT License | https://asciidoctor.org */
/* Uncomment the following line when using as a custom stylesheet */
/* @import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700"; */
html{font-family:sans-serif;-webkit-text-size-adjust:100%}
a{background:none}
a:focus{outline:thin dotted}
a:active,a:hover{outline:0}
h1{font-size:2em;margin:.67em 0}
b,strong{font-weight:bold}
abbr{font-size:.9em}
abbr[title]{cursor:help;border-bottom:1px dotted #dddddf;text-decoration:none}
dfn{font-style:italic}
hr{height:0}
mark{background:#ff0;color:#000}
code,kbd,pre,samp{font-family:monospace;font-size:1em}
pre{white-space:pre-wrap}
q{quotes:"\201C" "\201D" "\2018" "\2019"}
small{font-size:80%}
sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}
sup{top:-.5em}
sub{bottom:-.25em}
img{border:0}
svg:not(:root){overflow:hidden}
figure{margin:0}
audio,video{display:inline-block}
audio:not([controls]){display:none;height:0}
fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}
legend{border:0;padding:0}
button,input,select,textarea{font-family:inherit;font-size:100%;margin:0}
button,input{line-height:normal}
button,select{text-transform:none}
button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}
button[disabled],html input[disabled]{cursor:default}
input[type=checkbox],input[type=radio]{padding:0}
button::-moz-focus-inner,input::-moz-focus-inner{border:0;padding:0}
textarea{overflow:auto;vertical-align:top}
table{border-collapse:collapse;border-spacing:0}
*,::before,::after{box-sizing:border-box}
html,body{font-size:100%}
body{background:#fff;color:rgba(0,0,0,.8);padding:0;margin:0;font-family:"Noto Serif","DejaVu Serif",serif;line-height:1;position:relative;cursor:auto;-moz-tab-size:4;-o-tab-size:4;tab-size:4;word-wrap:anywhere;-moz-osx-font-smoothing:grayscale;-webkit-font-smoothing:antialiased}
a:hover{cursor:pointer}
img,object,embed{max-width:100%;height:auto}
object,embed{height:100%}
img{-ms-interpolation-mode:bicubic}
.left{float:left!important}
.right{float:right!important}
.text-left{text-align:left!important}
.text-right{text-align:right!important}
.text-center{text-align:center!important}
.text-justify{text-align:justify!important}
.hide{display:none}
img,object,svg{display:inline-block;vertical-align:middle}
textarea{height:auto;min-height:50px}
select{width:100%}
.subheader,.admonitionblock td.content>.title,.audioblock>.title,.exampleblock>.title,.imageblock>.title,.listingblock>.title,.literalblock>.title,.stemblock>.title,.openblock>.title,.paragraph>.title,.quoteblock>.title,table.tableblock>.title,.verseblock>.title,.videoblock>.title,.dlist>.title,.olist>.title,.ulist>.title,.qlist>.title,.hdlist>.title{line-height:1.45;color:#7a2518;font-weight:400;margin-top:0;margin-bottom:.25em}
div,dl,dt,dd,ul,ol,li,h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6,pre,form,p,blockquote,th,td{margin:0;padding:0}
a{color:#2156a5;text-decoration:underline;line-height:inherit}
a:hover,a:focus{color:#1d4b8f}
a img{border:0}
p{line-height:1.6;margin-bottom:1.25em;text-rendering:optimizeLegibility}
p aside{font-size:.875em;line-height:1.35;font-style:italic}
h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{font-family:"Open Sans","DejaVu Sans",sans-serif;font-weight:300;font-style:normal;color:#ba3925;text-rendering:optimizeLegibility;margin-top:1em;margin-bottom:.5em;line-height:1.0125em}
h1 small,h2 small,h3 small,#toctitle small,.sidebarblock>.content>.title small,h4 small,h5 small,h6 small{font-size:60%;color:#e99b8f;line-height:0}
h1{font-size:2.125em}
h2{font-size:1.6875em}
h3,#toctitle,.sidebarblock>.content>.title{font-size:1.375em}
h4,h5{font-size:1.125em}
h6{font-size:1em}
hr{border:solid #dddddf;border-width:1px 0 0;clear:both;margin:1.25em 0 1.1875em}
em,i{font-style:italic;line-height:inherit}
strong,b{font-weight:bold;line-height:inherit}
small{font-size:60%;line-height:inherit}
code{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;font-weight:400;color:rgba(0,0,0,.9)}
ul,ol,dl{line-height:1.6;margin-bottom:1.25em;list-style-position:outside;font-family:inherit}
ul,ol{margin-left:1.5em}
ul li ul,ul li ol{margin-left:1.25em;margin-bottom:0}
ul.square li ul,ul.circle li ul,ul.disc li ul{list-style:inherit}
ul.square{list-style-type:square}
ul.circle{list-style-type:circle}
ul.disc{list-style-type:disc}
ol li ul,ol li ol{margin-left:1.25em;margin-bottom:0}
dl dt{margin-bottom:.3125em;font-weight:bold}
dl dd{margin-bottom:1.25em}
blockquote{margin:0 0 1.25em;padding:.5625em 1.25em 0 1.1875em;border-left:1px solid #ddd}
blockquote,blockquote p{line-height:1.6;color:rgba(0,0,0,.85)}
@media screen and (min-width:768px){h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{line-height:1.2}
h1{font-size:2.75em}
h2{font-size:2.3125em}
h3,#toctitle,.sidebarblock>.content>.title{font-size:1.6875em}
h4{font-size:1.4375em}}
table{background:#fff;margin-bottom:1.25em;border:1px solid #dedede;word-wrap:normal}
table thead,table tfoot{background:#f7f8f7}
table thead tr th,table thead tr td,table tfoot tr th,table tfoot tr td{padding:.5em .625em .625em;font-size:inherit;color:rgba(0,0,0,.8);text-align:left}
table tr th,table tr td{padding:.5625em .625em;font-size:inherit;color:rgba(0,0,0,.8)}
table tr.even,table tr.alt{background:#f8f8f7}
table thead tr th,table tfoot tr th,table tbody tr td,table tr td,table tfoot tr td{line-height:1.6}
h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{line-height:1.2;word-spacing:-.05em}
h1 strong,h2 strong,h3 strong,#toctitle strong,.sidebarblock>.content>.title strong,h4 strong,h5 strong,h6 strong{font-weight:400}
.center{margin-left:auto;margin-right:auto}
.stretch{width:100%}
.clearfix::before,.clearfix::after,.float-group::before,.float-group::after{content:" ";display:table}
.clearfix::after,.float-group::after{clear:both}
:not(pre).nobreak{word-wrap:normal}
:not(pre).nowrap{white-space:nowrap}
:not(pre).pre-wrap{white-space:pre-wrap}
:not(pre):not([class^=L])>code{font-size:.9375em;font-style:normal!important;letter-spacing:0;padding:.1em .5ex;word-spacing:-.15em;background:#f7f7f8;border-radius:4px;line-height:1.45;text-rendering:optimizeSpeed}
pre{color:rgba(0,0,0,.9);font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;line-height:1.45;text-rendering:optimizeSpeed}
pre code,pre pre{color:inherit;font-size:inherit;line-height:inherit}
pre>code{display:block}
pre.nowrap,pre.nowrap pre{white-space:pre;word-wrap:normal}
em em{font-style:normal}
strong strong{font-weight:400}
.keyseq{color:rgba(51,51,51,.8)}
kbd{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;display:inline-block;color:rgba(0,0,0,.8);font-size:.65em;line-height:1.45;background:#f7f7f7;border:1px solid #ccc;border-radius:3px;box-shadow:0 1px 0 rgba(0,0,0,.2),inset 0 0 0 .1em #fff;margin:0 .15em;padding:.2em .5em;vertical-align:middle;position:relative;top:-.1em;white-space:nowrap}
.keyseq kbd:first-child{margin-left:0}
.keyseq kbd:last-child{margin-right:0}
.menuseq,.menuref{color:#000}
.menuseq b:not(.caret),.menuref{font-weight:inherit}
.menuseq{word-spacing:-.02em}
.menuseq b.caret{font-size:1.25em;line-height:.8}
.menuseq i.caret{font-weight:bold;text-align:center;width:.45em}
b.button::before,b.button::after{position:relative;top:-1px;font-weight:400}
b.button::before{content:"[";padding:0 3px 0 2px}
b.button::after{content:"]";padding:0 2px 0 3px}
p a>code:hover{color:rgba(0,0,0,.9)}
#header,#content,#footnotes,#footer{width:100%;margin:0 auto;max-width:62.5em;*zoom:1;position:relative;padding-left:.9375em;padding-right:.9375em}
#header::before,#header::after,#content::before,#content::after,#footnotes::before,#footnotes::after,#footer::before,#footer::after{content:" ";display:table}
#header::after,#content::after,#footnotes::after,#footer::after{clear:both}
#content{margin-top:1.25em}
#content::before{content:none}
#header>h1:first-child{color:rgba(0,0,0,.85);margin-top:2.25rem;margin-bottom:0}
#header>h1:first-child+#toc{margin-top:8px;border-top:1px solid #dddddf}
#header>h1:only-child,body.toc2 #header>h1:nth-last-child(2){border-bottom:1px solid #dddddf;padding-bottom:8px}
#header .details{border-bottom:1px solid #dddddf;line-height:1.45;padding-top:.25em;padding-bottom:.25em;padding-left:.25em;color:rgba(0,0,0,.6);display:flex;flex-flow:row wrap}
#header .details span:first-child{margin-left:-.125em}
#header .details span.email a{color:rgba(0,0,0,.85)}
#header .details br{display:none}
#header .details br+span::before{content:"\00a0\2013\00a0"}
#header .details br+span.author::before{content:"\00a0\22c5\00a0";color:rgba(0,0,0,.85)}
#header .details br+span#revremark::before{content:"\00a0|\00a0"}
#header #revnumber{text-transform:capitalize}
#header #revnumber::after{content:"\00a0"}
#content>h1:first-child:not([class]){color:rgba(0,0,0,.85);border-bottom:1px solid #dddddf;padding-bottom:8px;margin-top:0;padding-top:1rem;margin-bottom:1.25rem}
#toc{border-bottom:1px solid #e7e7e9;padding-bottom:.5em}
#toc>ul{margin-left:.125em}
#toc ul.sectlevel0>li>a{font-style:italic}
#toc ul.sectlevel0 ul.sectlevel1{margin:.5em 0}
#toc ul{font-family:"Open Sans","DejaVu Sans",sans-serif;list-style-type:none}
#toc li{line-height:1.3334;margin-top:.3334em}
#toc a{text-decoration:none}
#toc a:active{text-decoration:underline}
#toctitle{color:#7a2518;font-size:1.2em}
@media screen and (min-width:768px){#toctitle{font-size:1.375em}
body.toc2{padding-left:15em;padding-right:0}
#toc.toc2{margin-top:0!important;background:#f8f8f7;position:fixed;width:15em;left:0;top:0;border-right:1px solid #e7e7e9;border-top-width:0!important;border-bottom-width:0!important;z-index:1000;padding:1.25em 1em;height:100%;overflow:auto}
#toc.toc2 #toctitle{margin-top:0;margin-bottom:.8rem;font-size:1.2em}
#toc.toc2>ul{font-size:.9em;margin-bottom:0}
#toc.toc2 ul ul{margin-left:0;padding-left:1em}
#toc.toc2 ul.sectlevel0 ul.sectlevel1{padding-left:0;margin-top:.5em;margin-bottom:.5em}
body.toc2.toc-right{padding-left:0;padding-right:15em}
body.toc2.toc-right #toc.toc2{border-right-width:0;border-left:1px solid #e7e7e9;left:auto;right:0}}
@media screen and (min-width:1280px){body.toc2{padding-left:20em;padding-right:0}
#toc.toc2{width:20em}
#toc.toc2 #toctitle{font-size:1.375em}
#toc.toc2>ul{font-size:.95em}
#toc.toc2 ul ul{padding-left:1.25em}
body.toc2.toc-right{padding-left:0;padding-right:20em}}
#content #toc{border:1px solid #e0e0dc;margin-bottom:1.25em;padding:1.25em;background:#f8f8f7;border-radius:4px}
#content #toc>:first-child{margin-top:0}
#content #toc>:last-child{margin-bottom:0}
#footer{max-width:none;background:rgba(0,0,0,.8);padding:1.25em}
#footer-text{color:hsla(0,0%,100%,.8);line-height:1.44}
#content{margin-bottom:.625em}
.sect1{padding-bottom:.625em}
@media screen and (min-width:768px){#content{margin-bottom:1.25em}
.sect1{padding-bottom:1.25em}}
.sect1:last-child{padding-bottom:0}
.sect1+.sect1{border-top:1px solid #e7e7e9}
#content h1>a.anchor,h2>a.anchor,h3>a.anchor,#toctitle>a.anchor,.sidebarblock>.content>.title>a.anchor,h4>a.anchor,h5>a.anchor,h6>a.anchor{position:absolute;z-index:1001;width:1.5ex;margin-left:-1.5ex;display:block;text-decoration:none!important;visibility:hidden;text-align:center;font-weight:400}
#content h1>a.anchor::before,h2>a.anchor::before,h3>a.anchor::before,#toctitle>a.anchor::before,.sidebarblock>.content>.title>a.anchor::before,h4>a.anchor::before,h5>a.anchor::before,h6>a.anchor::before{content:"\00A7";font-size:.85em;display:block;padding-top:.1em}
#content h1:hover>a.anchor,#content h1>a.anchor:hover,h2:hover>a.anchor,h2>a.anchor:hover,h3:hover>a.anchor,#toctitle:hover>a.anchor,.sidebarblock>.content>.title:hover>a.anchor,h3>a.anchor:hover,#toctitle>a.anchor:hover,.sidebarblock>.content>.title>a.anchor:hover,h4:hover>a.anchor,h4>a.anchor:hover,h5:hover>a.anchor,h5>a.anchor:hover,h6:hover>a.anchor,h6>a.anchor:hover{visibility:visible}
#content h1>a.link,h2>a.link,h3>a.link,#toctitle>a.link,.sidebarblock>.content>.title>a.link,h4>a.link,h5>a.link,h6>a.link{color:#ba3925;text-decoration:none}
#content h1>a.link:hover,h2>a.link:hover,h3>a.link:hover,#toctitle>a.link:hover,.sidebarblock>.content>.title>a.link:hover,h4>a.link:hover,h5>a.link:hover,h6>a.link:hover{color:#a53221}
details,.audioblock,.imageblock,.literalblock,.listingblock,.stemblock,.videoblock{margin-bottom:1.25em}
details{margin-left:1.25rem}
details>summary{cursor:pointer;display:block;position:relative;line-height:1.6;margin-bottom:.625rem;-webkit-tap-highlight-color:transparent}
details>summary::before{content:"";border:solid transparent;border-left:solid;border-width:.3em 0 .3em .5em;position:absolute;top:.5em;left:-1.25rem;transform:translateX(15%)}
details[open]>summary::before{border:solid transparent;border-top:solid;border-width:.5em .3em 0;transform:translateY(15%)}
details>summary::after{content:"";width:1.25rem;height:1em;position:absolute;top:.3em;left:-1.25rem}
.admonitionblock td.content>.title,.audioblock>.title,.exampleblock>.title,.imageblock>.title,.listingblock>.title,.literalblock>.title,.stemblock>.title,.openblock>.title,.paragraph>.title,.quoteblock>.title,table.tableblock>.title,.verseblock>.title,.videoblock>.title,.dlist>.title,.olist>.title,.ulist>.title,.qlist>.title,.hdlist>.title{text-rendering:optimizeLegibility;text-align:left;font-family:"Noto Serif","DejaVu Serif",serif;font-size:1rem;font-style:italic}
table.tableblock.fit-content>caption.title{white-space:nowrap;width:0}
.paragraph.lead>p,#preamble>.sectionbody>[class=paragraph]:first-of-type p{font-size:1.21875em;line-height:1.6;color:rgba(0,0,0,.85)}
.admonitionblock>table{border-collapse:separate;border:0;background:none;width:100%}
.admonitionblock>table td.icon{text-align:center;width:80px}
.admonitionblock>table td.icon img{max-width:none}
.admonitionblock>table td.icon .title{font-weight:bold;font-family:"Open Sans","DejaVu Sans",sans-serif;text-transform:uppercase}
.admonitionblock>table td.content{padding-left:1.125em;padding-right:1.25em;border-left:1px solid #dddddf;color:rgba(0,0,0,.6);word-wrap:anywhere}
.admonitionblock>table td.content>:last-child>:last-child{margin-bottom:0}
.exampleblock>.content{border:1px solid #e6e6e6;margin-bottom:1.25em;padding:1.25em;background:#fff;border-radius:4px}
.exampleblock>.content>:first-child{margin-top:0}
.exampleblock>.content>:last-child{margin-bottom:0}
.sidebarblock{border:1px solid #dbdbd6;margin-bottom:1.25em;padding:1.25em;background:#f3f3f2;border-radius:4px}
.sidebarblock>:first-child{margin-top:0}
.sidebarblock>:last-child{margin-bottom:0}
.sidebarblock>.content>.title{color:#7a2518;margin-top:0;text-align:center}
.exampleblock>.content>:last-child>:last-child,.exampleblock>.content .olist>ol>li:last-child>:last-child,.exampleblock>.content .ulist>ul>li:last-child>:last-child,.exampleblock>.content .qlist>ol>li:last-child>:last-child,.sidebarblock>.content>:last-child>:last-child,.sidebarblock>.content .olist>ol>li:last-child>:last-child,.sidebarblock>.content .ulist>ul>li:last-child>:last-child,.sidebarblock>.content .qlist>ol>li:last-child>:last-child{margin-bottom:0}
.literalblock pre,.listingblock>.content>pre{border-radius:4px;overflow-x:auto;padding:1em;font-size:.8125em}
@media screen and (min-width:768px){.literalblock pre,.listingblock>.content>pre{font-size:.90625em}}
@media screen and (min-width:1280px){.literalblock pre,.listingblock>.content>pre{font-size:1em}}
.literalblock pre,.listingblock>.content>pre:not(.highlight),.listingblock>.content>pre[class=highlight],.listingblock>.content>pre[class^="highlight "]{background:#f7f7f8}
.literalblock.output pre{color:#f7f7f8;background:rgba(0,0,0,.9)}
.listingblock>.content{position:relative}
.listingblock code[data-lang]::before{display:none;content:attr(data-lang);position:absolute;font-size:.75em;top:.425rem;right:.5rem;line-height:1;text-transform:uppercase;color:inherit;opacity:.5}
.listingblock:hover code[data-lang]::before{display:block}
.listingblock.terminal pre .command::before{content:attr(data-prompt);padding-right:.5em;color:inherit;opacity:.5}
.listingblock.terminal pre .command:not([data-prompt])::before{content:"$"}
.listingblock pre.highlightjs{padding:0}
.listingblock pre.highlightjs>code{padding:1em;border-radius:4px}
.listingblock pre.prettyprint{border-width:0}
.prettyprint{background:#f7f7f8}
pre.prettyprint .linenums{line-height:1.45;margin-left:2em}
pre.prettyprint li{background:none;list-style-type:inherit;padding-left:0}
pre.prettyprint li code[data-lang]::before{opacity:1}
pre.prettyprint li:not(:first-child) code[data-lang]::before{display:none}
table.linenotable{border-collapse:separate;border:0;margin-bottom:0;background:none}
table.linenotable td[class]{color:inherit;vertical-align:top;padding:0;line-height:inherit;white-space:normal}
table.linenotable td.code{padding-left:.75em}
table.linenotable td.linenos{border-right:1px solid;opacity:.35;padding-right:.5em}
pre.pygments .lineno{border-right:1px solid;opacity:.35;display:inline-block;margin-right:.75em}
pre.pygments .lineno::before{content:"";margin-right:-.125em}
.quoteblock{margin:0 1em 1.25em 1.5em;display:table}
.quoteblock:not(.excerpt)>.title{margin-left:-1.5em;margin-bottom:.75em}
.quoteblock blockquote,.quoteblock p{color:rgba(0,0,0,.85);font-size:1.15rem;line-height:1.75;word-spacing:.1em;letter-spacing:0;font-style:italic;text-align:justify}
.quoteblock blockquote{margin:0;padding:0;border:0}
.quoteblock blockquote::before{content:"\201c";float:left;font-size:2.75em;font-weight:bold;line-height:.6em;margin-left:-.6em;color:#7a2518;text-shadow:0 1px 2px rgba(0,0,0,.1)}
.quoteblock blockquote>.paragraph:last-child p{margin-bottom:0}
.quoteblock .attribution{margin-top:.75em;margin-right:.5ex;text-align:right}
.verseblock{margin:0 1em 1.25em}
.verseblock pre{font-family:"Open Sans","DejaVu Sans",sans-serif;font-size:1.15rem;color:rgba(0,0,0,.85);font-weight:300;text-rendering:optimizeLegibility}
.verseblock pre strong{font-weight:400}
.verseblock .attribution{margin-top:1.25rem;margin-left:.5ex}
.quoteblock .attribution,.verseblock .attribution{font-size:.9375em;line-height:1.45;font-style:italic}
.quoteblock .attribution br,.verseblock .attribution br{display:none}
.quoteblock .attribution cite,.verseblock .attribution cite{display:block;letter-spacing:-.025em;color:rgba(0,0,0,.6)}
.quoteblock.abstract blockquote::before,.quoteblock.excerpt blockquote::before,.quoteblock .quoteblock blockquote::before{display:none}
.quoteblock.abstract blockquote,.quoteblock.abstract p,.quoteblock.excerpt blockquote,.quoteblock.excerpt p,.quoteblock .quoteblock blockquote,.quoteblock .quoteblock p{line-height:1.6;word-spacing:0}
.quoteblock.abstract{margin:0 1em 1.25em;display:block}
.quoteblock.abstract>.title{margin:0 0 .375em;font-size:1.15em;text-align:center}
.quoteblock.excerpt>blockquote,.quoteblock .quoteblock{padding:0 0 .25em 1em;border-left:.25em solid #dddddf}
.quoteblock.excerpt,.quoteblock .quoteblock{margin-left:0}
.quoteblock.excerpt blockquote,.quoteblock.excerpt p,.quoteblock .quoteblock blockquote,.quoteblock .quoteblock p{color:inherit;font-size:1.0625rem}
.quoteblock.excerpt .attribution,.quoteblock .quoteblock .attribution{color:inherit;font-size:.85rem;text-align:left;margin-right:0}
p.tableblock:last-child{margin-bottom:0}
td.tableblock>.content{margin-bottom:1.25em;word-wrap:anywhere}
td.tableblock>.content>:last-child{margin-bottom:-1.25em}
table.tableblock,th.tableblock,td.tableblock{border:0 solid #dedede}
table.grid-all>*>tr>*{border-width:1px}
table.grid-cols>*>tr>*{border-width:0 1px}
table.grid-rows>*>tr>*{border-width:1px 0}
table.frame-all{border-width:1px}
table.frame-ends{border-width:1px 0}
table.frame-sides{border-width:0 1px}
table.frame-none>colgroup+*>:first-child>*,table.frame-sides>colgroup+*>:first-child>*{border-top-width:0}
table.frame-none>:last-child>:last-child>*,table.frame-sides>:last-child>:last-child>*{border-bottom-width:0}
table.frame-none>*>tr>:first-child,table.frame-ends>*>tr>:first-child{border-left-width:0}
table.frame-none>*>tr>:last-child,table.frame-ends>*>tr>:last-child{border-right-width:0}
table.stripes-all tr,table.stripes-odd tr:nth-of-type(odd),table.stripes-even tr:nth-of-type(even),table.stripes-hover tr:hover{background:#f8f8f7}
th.halign-left,td.halign-left{text-align:left}
th.halign-right,td.halign-right{text-align:right}
th.halign-center,td.halign-center{text-align:center}
th.valign-top,td.valign-top{vertical-align:top}
th.valign-bottom,td.valign-bottom{vertical-align:bottom}
th.valign-middle,td.valign-middle{vertical-align:middle}
table thead th,table tfoot th{font-weight:bold}
tbody tr th{background:#f7f8f7}
tbody tr th,tbody tr th p,tfoot tr th,tfoot tr th p{color:rgba(0,0,0,.8);font-weight:bold}
p.tableblock>code:only-child{background:none;padding:0}
p.tableblock{font-size:1em}
ol{margin-left:1.75em}
ul li ol{margin-left:1.5em}
dl dd{margin-left:1.125em}
dl dd:last-child,dl dd:last-child>:last-child{margin-bottom:0}
ol>li p,ul>li p,ul dd,ol dd,.olist .olist,.ulist .ulist,.ulist .olist,.olist .ulist{margin-bottom:.625em}
ul.checklist,ul.none,ol.none,ul.no-bullet,ol.no-bullet,ol.unnumbered,ul.unstyled,ol.unstyled{list-style-type:none}
ul.no-bullet,ol.no-bullet,ol.unnumbered{margin-left:.625em}
ul.unstyled,ol.unstyled{margin-left:0}
ul.checklist>li>p:first-child{margin-left:-1em}
ul.checklist>li>p:first-child>.fa-square-o:first-child,ul.checklist>li>p:first-child>.fa-check-square-o:first-child{width:1.25em;font-size:.8em;position:relative;bottom:.125em}
ul.checklist>li>p:first-child>input[type=checkbox]:first-child{margin-right:.25em}
ul.inline{display:flex;flex-flow:row wrap;list-style:none;margin:0 0 .625em -1.25em}
ul.inline>li{margin-left:1.25em}
.unstyled dl dt{font-weight:400;font-style:normal}
ol.arabic{list-style-type:decimal}
ol.decimal{list-style-type:decimal-leading-zero}
ol.loweralpha{list-style-type:lower-alpha}
ol.upperalpha{list-style-type:upper-alpha}
ol.lowerroman{list-style-type:lower-roman}
ol.upperroman{list-style-type:upper-roman}
ol.lowergreek{list-style-type:lower-greek}
.hdlist>table,.colist>table{border:0;background:none}
.hdlist>table>tbody>tr,.colist>table>tbody>tr{background:none}
td.hdlist1,td.hdlist2{vertical-align:top;padding:0 .625em}
td.hdlist1{font-weight:bold;padding-bottom:1.25em}
td.hdlist2{word-wrap:anywhere}
.literalblock+.colist,.listingblock+.colist{margin-top:-.5em}
.colist td:not([class]):first-child{padding:.4em .75em 0;line-height:1;vertical-align:top}
.colist td:not([class]):first-child img{max-width:none}
.colist td:not([class]):last-child{padding:.25em 0}
.thumb,.th{line-height:0;display:inline-block;border:4px solid #fff;box-shadow:0 0 0 1px #ddd}
.imageblock.left{margin:.25em .625em 1.25em 0}
.imageblock.right{margin:.25em 0 1.25em .625em}
.imageblock>.title{margin-bottom:0}
.imageblock.thumb,.imageblock.th{border-width:6px}
.imageblock.thumb>.title,.imageblock.th>.title{padding:0 .125em}
.image.left,.image.right{margin-top:.25em;margin-bottom:.25em;display:inline-block;line-height:0}
.image.left{margin-right:.625em}
.image.right{margin-left:.625em}
a.image{text-decoration:none;display:inline-block}
a.image object{pointer-events:none}
sup.footnote,sup.footnoteref{font-size:.875em;position:static;vertical-align:super}
sup.footnote a,sup.footnoteref a{text-decoration:none}
sup.footnote a:active,sup.footnoteref a:active{text-decoration:underline}
#footnotes{padding-top:.75em;padding-bottom:.75em;margin-bottom:.625em}
#footnotes hr{width:20%;min-width:6.25em;margin:-.25em 0 .75em;border-width:1px 0 0}
#footnotes .footnote{padding:0 .375em 0 .225em;line-height:1.3334;font-size:.875em;margin-left:1.2em;margin-bottom:.2em}
#footnotes .footnote a:first-of-type{font-weight:bold;text-decoration:none;margin-left:-1.05em}
#footnotes .footnote:last-of-type{margin-bottom:0}
#content #footnotes{margin-top:-.625em;margin-bottom:0;padding:.75em 0}
.gist .file-data>table{border:0;background:#fff;width:100%;margin-bottom:0}
.gist .file-data>table td.line-data{width:99%}
div.unbreakable{page-break-inside:avoid}
.big{font-size:larger}
.small{font-size:smaller}
.underline{text-decoration:underline}
.overline{text-decoration:overline}
.line-through{text-decoration:line-through}
.aqua{color:#00bfbf}
.aqua-background{background:#00fafa}
.black{color:#000}
.black-background{background:#000}
.blue{color:#0000bf}
.blue-background{background:#0000fa}
.fuchsia{color:#bf00bf}
.fuchsia-background{background:#fa00fa}
.gray{color:#606060}
.gray-background{background:#7d7d7d}
.green{color:#006000}
.green-background{background:#007d00}
.lime{color:#00bf00}
.lime-background{background:#00fa00}
.maroon{color:#600000}
.maroon-background{background:#7d0000}
.navy{color:#000060}
.navy-background{background:#00007d}
.olive{color:#606000}
.olive-background{background:#7d7d00}
.purple{color:#600060}
.purple-background{background:#7d007d}
.red{color:#bf0000}
.red-background{background:#fa0000}
.silver{color:#909090}
.silver-background{background:#bcbcbc}
.teal{color:#006060}
.teal-background{background:#007d7d}
.white{color:#bfbfbf}
.white-background{background:#fafafa}
.yellow{color:#bfbf00}
.yellow-background{background:#fafa00}
span.icon>.fa{cursor:default}
a span.icon>.fa{cursor:inherit}
.admonitionblock td.icon [class^="fa icon-"]{font-size:2.5em;text-shadow:1px 1px 2px rgba(0,0,0,.5);cursor:default}
.admonitionblock td.icon .icon-note::before{content:"\f05a";color:#19407c}
.admonitionblock td.icon .icon-tip::before{content:"\f0eb";text-shadow:1px 1px 2px rgba(155,155,0,.8);color:#111}
.admonitionblock td.icon .icon-warning::before{content:"\f071";color:#bf6900}
.admonitionblock td.icon .icon-caution::before{content:"\f06d";color:#bf3400}
.admonitionblock td.icon .icon-important::before{content:"\f06a";color:#bf0000}
.conum[data-value]{display:inline-block;color:#fff!important;background:rgba(0,0,0,.8);border-radius:50%;text-align:center;font-size:.75em;width:1.67em;height:1.67em;line-height:1.67em;font-family:"Open Sans","DejaVu Sans",sans-serif;font-style:normal;font-weight:bold}
.conum[data-value] *{color:#fff!important}
.conum[data-value]+b{display:none}
.conum[data-value]::after{content:attr(data-value)}
pre .conum[data-value]{position:relative;top:-.125em}
b.conum *{color:inherit!important}
.conum:not([data-value]):empty{display:none}
dt,th.tableblock,td.content,div.footnote{text-rendering:optimizeLegibility}
h1,h2,p,td.content,span.alt,summary{letter-spacing:-.01em}
p strong,td.content strong,div.footnote strong{letter-spacing:-.005em}
p,blockquote,dt,td.content,span.alt,summary{font-size:1.0625rem}
p{margin-bottom:1.25rem}
.sidebarblock p,.sidebarblock dt,.sidebarblock td.content,p.tableblock{font-size:1em}
.exampleblock>.content{background:#fffef7;border-color:#e0e0dc;box-shadow:0 1px 4px #e0e0dc}
.print-only{display:none!important}
@page{margin:1.25cm .75cm}
@media print{*{box-shadow:none!important;text-shadow:none!important}
html{font-size:80%}
a{color:inherit!important;text-decoration:underline!important}
a.bare,a[href^="#"],a[href^="mailto:"]{text-decoration:none!important}
a[href^="http:"]:not(.bare)::after,a[href^="https:"]:not(.bare)::after{content:"(" attr(href) ")";display:inline-block;font-size:.875em;padding-left:.25em}
abbr[title]{border-bottom:1px dotted}
abbr[title]::after{content:" (" attr(title) ")"}
pre,blockquote,tr,img,object,svg{page-break-inside:avoid}
thead{display:table-header-group}
svg{max-width:100%}
p,blockquote,dt,td.content{font-size:1em;orphans:3;widows:3}
h2,h3,#toctitle,.sidebarblock>.content>.title{page-break-after:avoid}
#header,#content,#footnotes,#footer{max-width:none}
#toc,.sidebarblock,.exampleblock>.content{background:none!important}
#toc{border-bottom:1px solid #dddddf!important;padding-bottom:0!important}
body.book #header{text-align:center}
body.book #header>h1:first-child{border:0!important;margin:2.5em 0 1em}
body.book #header .details{border:0!important;display:block;padding:0!important}
body.book #header .details span:first-child{margin-left:0!important}
body.book #header .details br{display:block}
body.book #header .details br+span::before{content:none!important}
body.book #toc{border:0!important;text-align:left!important;padding:0!important;margin:0!important}
body.book #toc,body.book #preamble,body.book h1.sect0,body.book .sect1>h2{page-break-before:always}
.listingblock code[data-lang]::before{display:block}
#footer{padding:0 .9375em}
.hide-on-print{display:none!important}
.print-only{display:block!important}
.hide-for-print{display:none!important}
.show-for-print{display:inherit!important}}
@media amzn-kf8,print{#header>h1:first-child{margin-top:1.25rem}
.sect1{padding:0!important}
.sect1+.sect1{border:0}
#footer{background:none}
#footer-text{color:rgba(0,0,0,.6);font-size:.9em}}
@media amzn-kf8{#header,#content,#footnotes,#footer{padding:0}}
</style>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<style>
    div.ulist li {
        margin-bottom: -0.6em;
    }
    div.ulist li p {
        line-height: 1.2em;
    }
    div.ulist ul {
        margin-top: -0.5em;
        margin-bottom: 2.5em;
    }
    div.sect1 {
        margin-bottom: -1.5em;
    }
    h2 {
        margin-top: 0.5em;
    }
    body {
        font-family: Ubuntu, sans-serif;
    }
</style>
</head>
<body class="article toc2 toc-left">
<div id="header">
<h1><strong>Francesco Ballesio</strong></h1>
<div class="details">
<span id="author" class="author">Francesco Ballesio Ph.D.</span><br>
<span id="email" class="email"><a href="mailto:francesco.ballesio@uniroma2.it">francesco.ballesio[at]uniroma2.it</a></span><br>
</div>
<div id="toc" class="toc2">
<div id="toctitle">CV content</div>
<ul class="sectlevel1">
<li><a href="#_personal_details"><span class="icon"><i class="fa fa-user"></i></span> Personal details</a></li>
<li><a href="#_current_position"><span class="icon"><i class="fa fa-black-tie"></i></span> Current position</a></li>
<li><a href="#_collaborations"><span class="icon"><i class="fa fa-search"></i></span> Collaborations</a></li>
<li><a href="#_employment"><span class="icon"><i class="fa fa-suitcase"></i></span> Employment</a></li>
<li><a href="#_education"><span class="icon"><i class="fa fa-graduation-cap"></i></span> Education</a></li>
<li><a href="#_teaching"><span class="icon"><i class="fa fa-users"></i></span> Teaching</a></li>
<li><a href="#_symposia"><span class="icon"><i class="fa fa-calendar"></i></span> Symposia</a></li>
<li><a href="#_mentoring"><span class="icon"><i class="fa fa-handshake-o"></i></span> Mentoring</a></li>
<li><a href="#_invitations"><span class="icon"><i class="fa fa-microphone"></i></span> Invitations</a></li>
<li><a href="#_publications"><span class="icon"><i class="fa fa-book"></i></span> Publications</a></li>
<li><a href="#_research_experience"><span class="icon"><i class="fa fa-flask"></i></span> Research experience</a></li>
<li><a href="#_awards"><span class="icon"><i class="fa fa-dollar"></i></span> Awards</a></li>
<li><a href="#_talks_posters"><span class="icon"><i class="fa fa-bullhorn"></i></span> Talks &amp; Posters</a></li>
<li><a href="#_courses"><span class="icon"><i class="fa fa-pencil"></i></span> Courses</a></li>
<li><a href="#_misc"><span class="icon"><i class="fa fa-cloud"></i></span> Misc</a></li>
</ul>
</div>
</div>
<div id="content">
<div class="sect1">
<h2 id="_personal_details"><span class="icon"><i class="fa fa-user"></i></span> Personal details</h2>
<div class="sectionbody">
<div class="imageblock">
<div class="content">
<img src="images/francesco_ballesio.jpg" alt="FrancescoBallesioPhoto" width="200">
</div>
</div>
<div class="paragraph">
<p>Postdoctoral fellow in Bioinformatics</p>
</div>

<div class="paragraph">
<p><span class="icon"><i class="fa fa-linkedin"></i></span> <a href="https://www.linkedin.com/in/francesco-ballesio">Linkedin</a> |
<span class="icon"><i class="fa fa-google"></i></span> <a href="https://scholar.google.it/citations?user=UnndWJwAAAAJ&hl=en&oi=ao">Papers</a> |
<span class="icon"><i class="fa fa-registered"></i></span> <a href="https://www.researchgate.net/profile/Francesco-Ballesio">ResearchGate</a> |
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/francescoballesio">Code</a> |
<span class="icon"><i class="fa fa-address-card"></i></span> <a href="https://orcid.org/0000-0002-5892-5490">ORCID</a> |
</p>
</div>
</div>
</div>
    
<div class="sect1">
<h2 id="_current_position"><span class="icon"><i class="fa fa-black-tie"></i></span> Current position</h2>
<div class="sectionbody">
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>15 Mar. 2023 to 15 Jan. 2026</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Postdoctoral Researcher at the <a href="https://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-group"></i></span> Reporting to Prof. Manuela Helmer-Citterich and Prof. Pier Federico Gherardini.</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Development of bioinformatics methods for RNA omics data analysis (<span class="icon"><i class="fa fa-book"></i></span>)</p>
</li>
</ul>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_collaborations"><span class="icon"><i class="fa fa-search"></i></span> Collaborations</h2>
<div class="sectionbody">
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Sep. 2022 to Present</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong><a href="https://zappasodilab.weill.cornell.edu/">Zappasodi lab</a> at <a href="https://weill.cornell.edu/">Weill Cornell Medical College</a></strong>  (New York, USA).<br>
<span class="icon"><i class="fa fa-group"></i></span> Reporting to Prof. Roberta Zappasodi.</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Effects of glycolysis alteration on tumors and their microenvironment</p>
</li>
<li>
<p>Analysis of bulk RNA-seq data</p>
</li>
<li>
<p>Analysis of Assay for Transposase-Accessible Chromatin using sequencing (ATAC-seq) data</p>
</li>
<li>
<p>Analysis of single cell and spatial transcriptomics data</p>
</li>
</ul>
</div>
    
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Sep. 2024 to Present</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Clinical Medicine and Hepatology <a href="https://www.campusbiomedicohospital.com/">Policlinico Universitario Campus Bio-Medico</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-group"></i></span> Reporting to Prof. Umberto Vespasiani Gentilucci.</p>
</div>
</div> 
</div>

<div class="sect1">
<h2 id="_employment"><span class="icon"><i class="fa fa-suitcase"></i></span> Employment</h2>
<div class="sectionbody">
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Nov. 2023 to Present</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Teacher of Computational Genomics - <a href="https://lmbioinfo.bio.uniroma2.it/">MSc in Bioinformatics</a> - <a href="https://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
</div>

<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>15 Mar. 2023 to 15 Jan. 2026</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Postdoctoral Researcher at the <a href="https://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-group"></i></span> Reporting to Prof. Manuela Helmer-Citterich and Prof. Pier Federico Gherardini.</p>
</div>

<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>15 Nov. 2022 to 15 Mar. 2023</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Post-degree Scholarship at the <a href="https://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-group"></i></span> Reporting to Prof. Manuela Helmer-Citterich and Prof. Pier Federico Gherardini.</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Functional annotation of lncRNAs</p>
</li>
</ul>
</div>
    
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Jun. 2014 to 1 Nov. 2019</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Administrative assistant - <a href="https://www.bluvet.it/clinica/policlinico-veterinario-gregorio-vii/">Centro Medico Chirurgico Veterinario s.r.l.</a> (Veterinary clinic)</strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-group"></i></span> Reporing to Doc. Matteo Tommasini Degna.</p>
</div>
</div>
</div>

    
<div class="sect1">
<h2 id="_education"><span class="icon"><i class="fa fa-graduation-cap"></i></span> Education</h2>
<div class="sectionbody">
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Nov. 2019 to 2 Mar. 2023</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Ph.D. in Cellular and Molecular Biology (Bioinformatics), <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-group"></i></span> Supervisors: Prof. Manuela Helmer-Citterich and Prof. Andrea Novelletto.<br>
<span class="icon"><i class="fa fa-book"></i></span> Thesis: "Investigating the possibility of a modular organization in lncRNA genes".</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Member of the exam board commission for Molecular Biology and Bioinformatics (BSc) and Web Applications for Biomedicine, Gene Expression and Data Structures (MSc)</p>
</li>
</ul>
</div>
    
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Mar. 2018 to 25 15 Oct. 2019</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Master’s degree in Bioinformatics (LM-6), <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-book"></i></span> Thesis: "Natural language processing applied to the human exons for a study about the evolution of genes".<br>
<span class="icon"><i class="fa fa-group"></i></span> Supervisors: Prof. Manuela Helmer-Citterich and and Doc. Marco Pietrosanto.<br>
<span class="icon"><i class="fa fa-battery-full"></i></span> Evaluation: 110/110 cum laude; GPA: 4.00, A+.</p>
</div>

<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Sep. 2014 to 1 Mar. 2018</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Bachelor’s degree in Biological Sciences (L-13), <a href="https://web.uniroma2.it/en/contenuto/biological_sciences-88686">University of Rome Tor Vergata</a></strong> (Rome, Italy).<br>
<span class="icon"><i class="fa fa-book"></i></span> Thesis: "A study on the RNA secondary structure motifs of interaction with Zinc Fingers".<br>
<span class="icon"><i class="fa fa-group"></i></span> Supervisors: Prof. Manuela Helmer-Citterich and Doc. Marco Pietrosanto.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_teaching"><span class="icon"><i class="fa fa-users"></i></span> Teaching</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 17.2413%;">
<col style="width: 41.3793%;">
<col style="width: 27.5862%;">
<col style="width: 13.7932%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Course</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-pencil"></i></span> Role</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>4 Dec. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Methods in genomic variant calling training course at <a href="https://humantechnopole.it/en/">Human Technopole</a> (Milan, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Instructor</strong>, created new material</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://github.com/HTGenomeAnalysisUnit/varcall_training/tree/main/2024/slides">Website</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>21 Oct. 2024 to 25 Oct. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human Pangenome BYOD Analysis Workshop by <a href="https://www.h3abionet.org/">H3ABioNet</a> (Cape Town, South Africa)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Organizer</strong> and <strong>instructor</strong>, created new material and analyzed new human genome assemblies</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/pangenome/HumanPangenomeBYOD2024">Material</a><br>
<span class="icon"><i class="fa fa-book"></i></span> <a href="https://elwazi.org/trainings/28">Website</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>7 Oct. 2024 to 8 Oct. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Computational Pangenomics, <a href="https://www.afinsubria.org/">AFI - Alta Formazione Insubria</a> (Pozzolo, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Organizer</strong> and <strong>instructor</strong>, created new material and tutorials, held lessons, assisted the participants, configured the machines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://www.afinsubria.org/2024/07/09/computational-pangenomics/">Website</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/AndreaGuarracino/computational_pangenomics">Repository</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/CompPangenomics2024AFI_CertificateOrganizerInstructor_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>19 June 2024 to 21 June 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Computational Pangenomics, <a href="https://www.uniroma1.it/">University of Rome La Sapienza</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Organizer</strong> and <strong>instructor</strong>, created new material and tutorials, held lessons, assisted the participants, configured the machines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://elixir-iib-training.github.io/site/2024-06-19-Pangenomics">Website</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/RomePanG24_ElixirCourse_CertificateOrganizerInstructor_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>18 May 2024 to 22 May 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MemPanG24 Pangenomics, <a href="https://www.uthsc.edu/">University of Tennessee Health and Science Center</a> (Memphis, USA)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Organizer</strong>, <strong>instructor</strong> and <strong>chair</strong>, created new material and tutorials, held lessons, assisted the participants, configured the virtual machines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://pangenome.github.io/MemPanG24/">Website</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/pangenome/MemPanG24/">Material</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/MemPanG24_CertificateOrganizerInstructorChair_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>26 Oct. 2020 to Present</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Data Structures for Bioinformatics (2 CFU/ECTS), Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Teacher</strong>, created new material, held all the lessons, assisted the students, did final exams</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/AndreaGuarracino/DataStructuresForBioinformatics">Material</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>17 Apr. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PhD Program in Genetics, Molecular and Cellular Biology, <a href="http://phdsgb.unipv.eu/site/en/home.html">University of Pavia</a> (Pavia, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Teacher</strong>, introducing pangenomics and PGGB (PanGenome Graph Builder)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/UniPaviaPhDCoursePangenome2024_Flier_Apr18-19_2024_final_AndreaGuarracino.pdf">Flier</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/UniPaviaPhDCoursePangenome2024_InvitationMail_AndreaGuarracino.pdf">Invitation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>9 Apr. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Advanced Bioinformatics: Data Mining and Data Integration for Life Science (1.5 CFU/ECTS), Master’s degree, <a href="https://ubc.uu.nl/">Utrecht Bioinformatics Center</a> (Utrecht, Netherlands)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Teacher</strong>, introducing pangenomics and PGGB (PanGenome Graph Builder)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/UMCUGenetics/AdvancedBioinformaticsCourse">Website</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>13 Nov. 2023 to 17 Nov. 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome &amp; RefGraph Workshop by <a href="https://www.h3abionet.org/">H3ABioNet</a> (Cape Town, South Africa)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Instructor</strong>, created new material and analyzed new human genome assemblies</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://github.com/AndreaGuarracino/ReferenceGraphPangenomeDataAnalysisHackathon2023">Material</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/certificates/H3ABioNet2023_RefGraphInvitationLetter_AndreaGuarracino.pdf">Invitation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>8 Oct. 2023 to 12 Oct. 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Complex Trait Community &amp; Rat Genome <a href="https://complextrait.org/meetings/ctc-rg2023/program.html">CTC-RG2023</a> (Memphis, USA)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Organizer</strong> and <strong>instructor</strong>, created new material for the Pangenome workshop</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://complextrait.org/meetings/ctc-rg2023/program.html">Website</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/certificates/CTCTG2023_CertificateOrganizerInstructor_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>30 May 2023 to 2 June 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MemPanG23 Pangenomics, <a href="https://www.uthsc.edu/">University of Tennessee Health and Science Center</a> (Memphis, USA)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Organizer</strong>, <strong>instructor</strong> and <strong>chair</strong>, created new material and tutorials, held lessons, assisted the participants, configured the virtual machines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://pangenome.github.io/MemPanG23/">Website</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/pangenome/MemPanG23/">Material</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/MemPanG23_CertificateOrganizerInstructorChair_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>28 Feb. 2023 to 1 Mar. 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Advanced Bioinformatics: Data Mining and Data Integration for Life Science (1.5 CFU/ECTS), Master’s degree, <a href="https://ubc.uu.nl/">Utrecht Bioinformatics Center</a> (Utrecht, Netherlands)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Teacher</strong>, introducing pangenomics and PGGB (PanGenome Graph Builder)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/UMCUGenetics/AdvancedBioinformaticsCourse">Website</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>23 May 2022 to 27 May 2022</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">CPANG22 Computational PANGenomics, <a href="https://gulbenkian.pt/ciencia/">Instituto Gulbenkian de Ciência</a> (Oeiras, Portugal)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Instructor</strong>, created new material and tutorials, held some lessons, assisted the participants</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-github"></i></span> <a href="https://gtpb.github.io/CPANG22/">Website</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/GTPB/CPANG22">Material</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/other/CPANG22_Poster.AndreaGuarracino.pdf">Poster</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/certificates/CPANG22_CertificateInstructor.AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2018 to 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatics (6 CFU/ECTS), Bachelor’s degree in Biological Science, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Tutor</strong>, reviewed the practical lessons, assisted the students</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/cv/blob/main/other/AttestatoTutoratoBioinformatica.DigitalSign.AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>16 Dec. 2019 to 29 Mar. 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Computational Proteogenomics (2 CFU/ECTS), Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Exam assistant</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 1</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>7 June 2019 to 21 July 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Molecular Biology (8 CFU/ECTS), Bachelor’s degree in Biological Science, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Exam assistant</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 1</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>7 June 2019 to 21 July 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatics (6 CFU/ECTS), Bachelor’s degree in Biological Science, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Exam assistant</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 1</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_symposia"><span class="icon"><i class="fa fa-calendar"></i></span> Symposia</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 18.1818%;">
<col style="width: 36.3636%;">
<col style="width: 36.3636%;">
<col style="width: 9.091%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Event</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-pencil"></i></span> Role</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>7 July 2024 to 11 July 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://smbe2024.org/">Society for Molecular Biology &amp; Evolution 2024 (SMBE24)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Organizer of the "Human genetic variability in the Pangenomic era" symposia</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/images/SMBE24_CallForAbstracts_AndreaGuarracino.png">Call for abstracts</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/certificates/SMBE24_CertificateOfAttendance_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_mentoring"><span class="icon"><i class="fa fa-handshake-o"></i></span> Mentoring</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 18.1818%;">
<col style="width: 36.3636%;">
<col style="width: 36.3636%;">
<col style="width: 9.091%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-university"></i></span> University</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Thesis</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2 Dec. 2024 to Present.</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome-based pipelines for unbiased ancient DNA analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">-</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2 Oct. 2024 to Present.</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Benchmarking Analytical Pipelines for Single-Cell Long-Read Sequencing in Colon Cancer</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">-</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>1 Oct. 2023 to Present</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PhD program in Systems Medicine, Computational Biology curricula, <a href="https://www.semm.it/curriculum/computational-biology">SEMM European School of Molecular Medicine</a> (Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Dissecting the intracellular response to replication and mitotic stress in colorectal cancer for the design of novel effective (immuno)therapies</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">-</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2 Sept. 2021 to 20 Apr. 2022</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Machine learning to automatically detect the <em>C. elegans</em> bodies from microscope images and measure their growth over time</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 3</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>20 Apr. 2021 to 02 Nov. 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bachelor’s degree in Biological Science, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Analysis of the effect of SARS-CoV-2 mutations on the pairing between the viral genome and human miRNAs</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 3</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>28 Apr. 2020 to 5 Nov. 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bachelor’s degree in Biological Science, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Analysis of repeated sequences in lincRNA candidates for exon shuffling</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 3</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>9 Mar. 2020 to 12 Mar. 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Machine Learning methods applied to kinase-substrate interaction prediction</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 3</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2 Mar. 2020 to 17 Dec. 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Master’s degree in Bioinformatics, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Natural Language Processing techniques for protein encoding applied to phosphorylation prediction</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 3</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>13 Dec. 2019 to 11 June 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bachelor’s degree in Biological Science, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a> (Rome, Italy)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Study of normalization techniques' effects in gene expression correlation analyses</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/other/EsamiSvoltiLaureeDocentePerAnnoAccademico.pdf">Page 3</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_invitations"><span class="icon"><i class="fa fa-microphone"></i></span> Invitations</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 40%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-globe"></i></span> Place</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Description</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>24 Sept. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.stowers.org/">Stowers Institute</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Seminar on "Pangenomes everywhere"</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/20240924_SeminarBrochureStowersInstitute_AndreaGuarracino.pdf">Brochure</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>26 Feb. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.uniroma1.it/">University of Rome La Sapienza</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Seminar on "Heterologous recombination in the human pangenome"</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/20240226_InvitationLetterLaSapienza_AndreaGuarracino.pdf">Invitation</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/20240226_SeminarBrochureLaSapienza_AndreaGuarracino.pdf">Brochure</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>14 Sept. 2023 to 16 Sept. 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.associazionegeneticaitaliana.it/2023/05/14/joint-meeting-agi_simag-2023/">Joint Meeting AGI-SIMAG, Cortona, Italy</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Invited speaker on "Heterologous recombination in the human pangenome"</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/20230606_AGI-SIMAG-2023_InvitationSpeakerLetter_AndreaGuarracino.pdf">Invitation</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/AGI-SIMAG-2023_Brochure_AndreaGuarracino.pdf">Brochure</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>13 Sept. 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://web.unisa.it/en/home">University of Salerno</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Seminar on "Heterologous recombination in the human pangenome"</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/20230913_SeminarInvitationTorVergata_AndreaGuarracino.pdf">Invitation</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_publications"><span class="icon"><i class="fa fa-book"></i></span> Publications</h2>
<div class="sectionbody">
<div class="paragraph">
<p><strong>*first authorship</strong></p>
</div>
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 12.5%;">
<col style="width: 37.5%;">
<col style="width: 37.5%;">
<col style="width: 12.5%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-newspaper-o"></i></span> Journal</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Title</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-pencil"></i></span> Contribution</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>bioRxiv, <em>Submitted</em></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>The formation and propagation of human Robertsonian chromosomes</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Genetic, epigenetic, and pangenome analyses, figures, paper writing</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11463614/">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Accepted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Complete sequencing of ape genomes</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome alignment/Divergence</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11312596/">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Accepted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Genetic Modulation of Protein Expression in Rat Brain</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome graph building and analyses, paper editing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://doi.org/10.1101/2024.02.17.580840">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Submitted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Comparative population pangenomes reveal unexpected complexity and fitness effects of structural variants</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome analysis and visualization</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://doi.org/10.1101/2025.02.11.637762">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Submitted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The Platinum Pedigree: A long-read benchmark for genetic variants</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://doi.org/10.1101/2024.10.02.616333">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Submitted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Creating a biomedical knowledge base by addressing GPT inaccurate responses and benchmarking context</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Testing, paper editing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc11507946/">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Submitted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Epigenetic control and inheritance of rDNA arrays</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nanopore reads analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11451732/">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">bioRxiv, <em>Submitted</em></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The complete diploid reference genome of RPE-1 identifies human phased epigenetic landscapes</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Support for the analyses and figures, Figure 2, paper writing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-spinner"></i></span> <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10760208/">Preprint</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">iScience, 2025</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome reconstruction in rats enhances genotype-phenotype mapping and novel variant discovery</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome graph building and analyses, structural variant calling, paper writing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1016/j.isci.2025.111835">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nature, 2025</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Leveraging a phased pangenome for haplotype design of hybrid potato</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome graph graph construction and evaluation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41586-024-08476-9">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Journal of the National Cancer Institute, 2024</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Impact of radiotherapy dose, fractionation and immunotherapeutic partner in a mouse model of HR+ mammary carcinogenesis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatic analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://pubmed.ncbi.nlm.nih.gov/39661487/">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Genome Research, 2024</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Gapless assembly of complete human and plant chromosomes using only nanopore sequencing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human centromere analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://pubmed.ncbi.nlm.nih.gov/39505490/">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Nature Methods, 2024</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>*Building pangenome graphs</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Software development, experiments, paper writing, documentation, testing</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41592-024-02430-3">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatics, 2024</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cluster efficient pangenome graph construction with nf-core/pangenome</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software development, paper editing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/bioinformatics/btae609">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Genome Research, 2024</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">High-coverage nanopore sequencing of samples from the 1000 Genomes Project to build a comprehensive catalog of human genetic variation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome graph building and analyses, paper method section writing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="http://doi.org/10.1101/gr.279273.124">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nature, 2024</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Recurrent evolution and selection shape structural diversity at the amylase locus</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome graph analsysis, genome assembly, paper writing and editing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41586-024-07911-1">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">arXiv, 2024</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Rapid GPU-Based Pangenome Graph Layout</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenomic support</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.48550/arxiv.2409.00876">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Bioinformatics, 2024</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>*Pangenome graph layout by Path-Guided Stochastic Gradient Descent</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Algorithm implementation and evaluation, paper writing</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/bioinformatics/btae609">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Genome Biology, 2023</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A super-pangenome of the North American wild grape species</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Support for pangenome graph building and read mapping against the graph</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1186/s13059-023-03133-2">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nature, 2023</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The complete sequence of a human Y chromosome</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Variants and Liftover</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://www.nature.com/articles/s41586-023-06457-y">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Frontiers in Genetics, 2023</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome Graphs in Infectious Disease: A Comprehensive Genetic Variation Analysis of Neisseria Meningitidis leveraging Oxford Nanopore long reads</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenomic analyses and support, paper editing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.3389/fgene.2023.1225248">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Nature, 2023</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>*Recombination between heterologous human acrocentric chromosomes</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Pangenomic analyses, Figures from 1 to 5, paper writing</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41586-023-05976-y">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nature, 2023</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A draft human pangenome reference</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Paper editing, pangenome graph creation and visualization, population genetic analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41586-023-05896-x">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatics, 2023</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Optimal gap-affine alignment in O(s) space</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Testing, evaluation, Figure 2</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/bioinformatics/btad074">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatics, 2022</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Unbiased pangenome graphs</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Parallelization of a bottleneck, bug fixings, experiments in multiple species</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/bioinformatics/btac743">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nature, 2022</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Semi-automated assembly of high-quality diploid human reference genomes</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pangenome alignments, Jaccard, PCA and MHC analyses, Figure 2</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41586-022-05325-5">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nature Immunology, 2022</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Type I IFNs promote cancer cell stemness by triggering the epigenetic regulator KDM1B</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Figure 6.a, 7.a, and 7.b, contributed for Figure 6.c,
6.e, performed RNA-seq, ATAC-seq, microarray data analysis, correlation analyses,
TF-binding motif enrichment, survival analyses</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41590-022-01290-3">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Bioinformatics, 2022</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>*ODGI: understanding pangenome graphs</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Paper and documentation writing, figures and table, implemented several tools</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/bioinformatics/btac308">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Non-coding RNA Research, 2022</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Evaluation of potential miRNA sponge effects of SARS genomes in human</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Genome sequences collection, filtering, deduplication, alignment, variant calling</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1016/j.ncrna.2022.01.003">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Nucleic Acids Research, 2021</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>*BRIO: a web server for RNA sequence and structure motif scan</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Paper writing, figures and table, refactored and completed the webserver, datasets refinement</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/nar/gkab400">Paper</a><br>
<span class="icon"><i class="fa fa-globe"></i></span> <a href="http://brio.bio.uniroma2.it/">Webserver</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/helmercitterich-lab/BRIO">Repository</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Communications Biology, 2021</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Genome-wide bioinformatic analyses predict key host and viral factors in SARS-CoV-2 pathogenesis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Differential expression analyses, including batch effects exploration, GO/pathway enrichment analyses</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s42003-021-02095-0">Paper</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/vaguiarpulido/covid19-research.git">Repository</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cancers, 2021</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The Targeting of MRE11 or RAD51 Sensitizes Colorectal Cancer Stem Cells to CHK1 Inhibition</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Figure 1.A, helped in analyzing survival data</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.3390/cancers13081957">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell Death &amp; Disease, 2021</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Inhibition of HECT E3 ligases as potential therapy for COVID-19</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Figure 4, <em>in silico</em> 3D structures analysis</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41419-021-03513-1">Paper</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>NAR Genomics &amp; Bioinformatics, 2021</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>*Relative Information Gain: Shannon entropy-based measure of the relative structural conservation in RNA alignments</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Paper writing, contributed to Figure 2, made Figure 3, 4, and 5, and fixed, refactored and completed the framework</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1093/nargab/lqab007">Paper</a><br>
<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/helmercitterich-lab/RIG">Repository</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell Death &amp; Disease, 2021</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Control of replication stress and mitosis in colorectal cancer stem cells through the interplay of PARP1, MRE11 and RAD51</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bioinformatics support</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://doi.org/10.1038/s41418-020-00733-4">Paper</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_research_experience"><span class="icon"><i class="fa fa-flask"></i></span> Research experience</h2>
<div class="sectionbody">
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>1 Nov. 2018 to 8 Nov. 2022</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Bioinformatic analyses of multiomics data, <a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a></strong> (Rome, Italy).</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Germline and somatic variants analyses on Whole-Exome sequencing (WES) data: quality control, sample matching, rRNA contamination, read trimming and mapping, variant calling, and functional prediction</p>
</li>
<li>
<p>Microsatellite instability (MSI) status on paired tumor-normal and tumor-only WES data</p>
</li>
<li>
<p>Variant calling and differential expression analyses on RNA-seq data</p>
</li>
<li>
<p>Neoantigen prediction and prioritization by integrating WES data and RNA-seq data</p>
</li>
<li>
<p>Analysis of Assay for Transposase-Accessible Chromatin using sequencing (ATAC-seq) data</p>
</li>
<li>
<p>Normalization and correlation analysis of transcriptomic microarray data (Affymetrix)</p>
</li>
<li>
<p>Proteomic and phosphoproteomic analyses of Reverse Phase Protein microarray (RPPA) data</p>
</li>
<li>
<p>RNA structural characterization and conservation research (<a href="https://doi.org/10.1093/nargab/lqab007">Pietrosanto, Adinolfi, Guarracino <em>et al.</em>, 2021</a>)</p>
</li>
<li>
<p>Web server development for RNA sequence and structure motif scan (<a href="https://doi.org/10.1093/nar/gkab400">Guarracino <em>et al.</em>, 2021</a>)</p>
</li>
<li>
<p>Energetic and functional analyses of phosphorylations applied <em>in silico</em> on 3D structures of protein complexes</p>
</li>
<li>
<p>Cox modeling and survival analysis on patient cohorts (from TCGA and cBioPortal platforms)</p>
</li>
<li>
<p>Basic procedures on High Performance Computing (HPC) machines</p>
</li>
</ul>
</div>
<div class="paragraph">
<p><span class="icon"><i class="fa fa-calendar"></i></span> <code>5 Apr. 2020 to 21 Sept. 2022</code><br>
<span class="icon"><i class="fa fa-university"></i></span> <strong>Previous software development for pangenomics</strong>.</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Workflow development of a public sequence resource for on-the-fly analyses (<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/pubseq/bh20-seq-resource">bh20-seq-resource</a>)</p>
</li>
<li>
<p>Development of a pangenome graph browser (<span class="icon"><i class="fa fa-github"></i></span> <a href="https://github.com/graph-genome/graph-genome.github.io">Pantograph</a>)</p>
</li>
</ul>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_awards"><span class="icon"><i class="fa fa-dollar"></i></span> Awards</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 16.6666%;">
<col style="width: 16.6666%;">
<col style="width: 33.3333%;">
<col style="width: 16.6666%;">
<col style="width: 16.6669%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-globe"></i></span> Place</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Description</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-money"></i></span> Amount</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>5 July 2022</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://iggsy.org/">International Genome Graph Symposium 2022</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Student Travel Grant</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">600 CHF (632.36 EUR)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-image-o"></i></span> <a href="https://andreaguarracino.github.io/other/IGGSy2022_StudentTravelGrant_AndreaGuarracino.jpg">Certificate</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_talks_posters"><span class="icon"><i class="fa fa-bullhorn"></i></span> Talks &amp; Posters</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 40%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-globe"></i></span> Conference</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Title</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2 Oct. 2024 to 5 Oct. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://rgd.mcw.edu/rgdweb/ctc-rg2024/program.html">Complex Trait Community and the Rat Genomics 2024</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Multi-platform genome assembly of an SHR/OlaIpcv X BN-Lx/Cub F1 rat "trio"</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/CTCRG24_MultiPlatformSHRXBNLxF1Trio_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>5 Sept. 2024 to 6 Sept. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">HPRC 2024 Annual Meeting</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lessons from the amylase locus</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/HPRC24_LessonsFromAmylaseLocus_Presentation_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>3 Sept. 2024 to 4 Sept. 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://sites.google.com/ucsc.edu/t2t-f2f/home">Telomere-to-Telomere "Face-to-Face" (T2T-F2F) 2024</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Formation and propagation of human Robertsonian chromosomes</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/T2T24_FormationTransmissionRobChrs_Presentation_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>30 June 2024 to 4 July 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://iggsy.org/">International Genome Graph Symposium 2024</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The complete sequence of Robertsonian chromosomes</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/IGGSy2024_TheCompleteSeqOfHumanRobChromosomes_Presentation_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>7 May 2024 to 11 May 2024</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://meetings.cshl.edu/meetings.aspx?meet=GENOME&amp;year=24">The Biology of Genomes 2024</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Aligning pangenomes with hierarchical wavefront algorithm</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/BoG24_AligningPangenomesWithHierarchicalWFA_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>23 July 2023 to 27 July 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.smbe2023.org/">Society for Molecular Biology &amp; Evolution 2023 (SMBE23)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Building a Pangenome Graph for the HXB/BXH Recombinant Inbred Rat Strain Family: Enhanced Discovery of Complex Variants and Validation with Sanger Sequencing</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/SMBE23_BuildingPangenomeGraphRecombInbredRatStrainFamily_Poster_AndreaGuarracino.pdf">Poster</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/certificates/SMBE23_CertificateOfAttendance_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>29 Sept. 2022 to 30 Sept. 2022</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.complextrait.org">Complex Trait Community and the Rat Genomics 2022</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Initial effort in generating a rat pangenome</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/CTC_RG_2022_InitialEffortGenRatPangenome_Abstract_AndreaGuarracino.pdf">Abstract</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>4 July 2022 to 7 July 2022</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://iggsy.org/">International Genome Graph Symposium 2022</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Chromosome communities in the human pangenome</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/IGGSy2022_ChromosomeCommunitiesHumanPangenome_Presentation_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>2 Nov. 2021 to 4 Nov. 2022</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://elixir-europe.org/events/3d-bioinfo-2021-annual-meeting">3D-BioInfo 2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RNA secondary structure motifs involved in the interaction with RNA binding proteins</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/3DBioinfo2021_RNASecondaryStructMotifsInvolvedInTheInteractionWithRBPs_Abstract_AndreaGuarracino.pdf">Abstract</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>27 Sept. 2021 to 1 Oct. 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://enrolment.engage-powered.com/hinxtonhall/go/register.aspx">Biodiversity Genomics 2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Chromosome communities in the human pangenome</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/BiodiversityGenomics2021_ChromosomeCommunitiesHumanPangenome_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/BiodiversityGenomics2021_ChromosomeCommunitiesHumanPangenome_Presentation_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>21 Sept. 2021 to 24 Sept. 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://agi2021.centercongressi.com/programme.php">AGI2021 Congress</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A pangenome for the expanded BXD family of mice</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/AGI2021_APangenomeForTheExpBXDfamOfMice_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/AGI2021_APangenomeForTheExpBXDfamOfMice_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>6 Sept. 2021 to 8 Sept. 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://dechema.converia.de/frontend/index.php?folder_id=3138&amp;page_id=">German Conference on Bioinformatics 2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ODGI: scalable tools for pangenome graphs</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/GCB2021_ODGIScalableToolsForPangenomeGraphs_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/GCB2021_ODGIScalableToolsForPangenomeGraphs_Presentation_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>9 June 2021 to 12 June 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.eacr2021.org/">EACR 2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Control of replication stress and mitosis in cancer stem cells</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/EACR2021_ControlReplicationStressAndMitosisInCancerStemCells_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/EACR2021_ControlReplicationStressAndMitosisInCancerStemCells_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>20 May 2021 to 21 May 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="http://www.embl-hamburg.de/aboutus/alumni/events-networks/local-chapters/italy/48_genoa_2021/index.html">EMBL in Italy 2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Identification of RNA sequence and structure motifs for protein interaction</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/EMBLInItaly2021_IdentificationOfRNASeqAndStrMotifsForProteinInteraction_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>11 May 2021 to 14 May 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://meetings.cshl.edu/meetings.aspx?meet=GENOME&amp;year=21">The Biology of Genomes 2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Identification of RNA sequence and structure motifs for protein interaction</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/BoG2021_ThePangenomeGraphBuilder_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/BoG2021_ThePanGenomeGraphBuilder_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>11 May 2021 to 14 May 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.bbcc-meetings.it/program/">BBCC2021</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Population genomics analyses on pangenome graphs</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/BBCC2020_PopulationGenomicsAnalysesOnPangenomeGraph_ProgramAndAbstractBook.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/presentations/f1000research-326757.pdf">Presentation</a> (<a href="https://f1000research.com/slides/9-1338">f1000research link</a>)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>28 Sept. 2020 to 1 Oct. 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.gcpr-vmv-vcbm-2020.uni-tuebingen.de/">EG VCBM 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Graph Layout by Path-Guided Stochastic Gradient</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/EG_VCMB_GraphLayoutByPath-GuidedStochasticGradientDescent_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/EG_VCMB_GraphLayoutByPath-GuidedStochasticGradientDescent_Poster_Landscape_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>21 Sept. 2020 to 23 Sept. 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.t2t-hprc-2020conference.com/">T2T-HPRC-Virtual Conference 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Graph Layout by Path-Guided Stochastic Gradient</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/T2T_HPRC_GraphLayoutByPath-GuidedStochasticGradientDescent_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/T2T_HPRC_GraphLayoutByPath-GuidedStochasticGradientDescent_Poster_Portrait_AndreaGuarracino.pdf">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>21 Sept. 2020 to 23 Sept. 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.t2t-hprc-2020conference.com/">T2T-HPRC-Virtual Conference 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Scalable Variant Detection In Pangenome Models</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/T2T_HPRC_ScalableVariantDetectionInPangenomeModels_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/BBCC2020_ScalableVariantDetectionInPangenomeModels_Poster_AndreaGuarracino.pdf">Presentation</a><br>
<span class="icon"><i class="fa fa-link"></i></span> <a href="https://gsocgraph.blogspot.com/2020/08/final-week-recap-of-my-gsoc-experience.html">Blog</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>17 July 2020 to 25 July 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://bcc2020.sched.com/">BCC 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">COVID-19 PubSeq: Public SARS-CoV-2 Sequence Resource</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/BCC2020_COVID19_PubSeq_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/BCC2020_COVID19_PubSeq_Poster_AndreaGuarracino.pdf">Poster</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://bcc2020.sched.com/event/coLw/covid-19-pubseq-public-sars-cov-2-sequence-resource">Presentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>13 July 2020 to 16 July 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.iscb.org/ismb2020">ISMB 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pantograph: Scalable Interactive Graph Genome Visualization</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/ISMB2020_PantographBrowsablePangenomeVisualization_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/ISMB2020_PantographBrowsablePangenomeVisualization_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>13 July 2020 to 16 July 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.iscb.org/ismb2020">ISMB 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Semantic Variation Graphs - A Pangenome Ontology</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/ISMB2020_SemanticVariationGraphs_OntologiesForPangenomeGraphs_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/ISMB2020_SemanticVariationGraphs_OntologiesForPangenomeGraphs_Poster_AndreaGuarracino.pdf">Poster</a><br>
<span class="icon"><i class="fa fa-link"></i></span> <a href="https://www.iscb.org/ismb2020-general/ismb2020-award-winners#bio-poster">Best Poster Prize</a><br>
<span class="icon"><i class="fa fa-book"></i></span> <a href="https://publikationen.bibliothek.kit.edu/1000127608">Citation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>13 July 2020 to 16 July 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.iscb.org/ismb2020">ISMB 2020</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Comprehensive analysis of human SARS-CoV-2 infection and host-virus interaction</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/abstracts/ISMB2020_ComprehensiveAnalysisSARSCoV2_Abstract_AndreaGuarracino.pdf">Abstract</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://andreaguarracino.github.io/posters/ISMB2020_ComprehensiveAnalysisSARSCoV2_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_courses"><span class="icon"><i class="fa fa-pencil"></i></span> Courses</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 40%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-globe"></i></span> Place</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> Title</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>17 July 2023 to 21 July 2023</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.lorentzcenter.nl/">Lorentz Center, Leiden, Netherlands</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A multidisciplinary approach to epistasis detection</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/other/LorentzEpistatisWorkshop_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>08 July 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.enago.com/academy/how-to-overcome-challenges-in-academic-writing/">Enago academy, Virtual</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Overcoming Challenges in Academic Writing: Tips for Writing Articles and Grant Applications</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/certificates/ENAGO_CertificateOvercomingChallengesAcademicWriting_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>21 Apr. 2018</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">English language certification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">English Speaking Board (ESB) Level 1 (B2 CEFR), Pass with distinction</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/certificates/EnglishCertificateESOL_B2_CEFR_AndreaGuarracino.pdf">Certificate</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>27 Sept. 2017 to 29 Sept. 2017</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://web.unisa.it/en/home">University of Salerno</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Best practices for RNA-Seq data analysis - ELIXIR-IIB</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-book"></i></span> <a href="https://elixir-iib-training.github.io/website/2017/09/27/RNA-seq-Salerno.html">Website</a><br>
<span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/certificates/ELIXIR_IIB_CertificateBestPracticesForRNAseqDataAnalysis_AndreaGuarracino.pdf">Certificate</a><br>
<span class="icon"><i class="fa fa-link"></i></span> <a href="https://github.com/ELIXIR-IIB-training/RNASeq2017">Repository</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="_misc"><span class="icon"><i class="fa fa-cloud"></i></span> Misc</h2>
<div class="sectionbody">
<table class="tableblock frame-all grid-all stripes-even stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 40%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-calendar"></i></span> Time</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-globe"></i></span> Info</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-book"></i></span> What</th>
<th class="tableblock halign-center valign-top"><span class="icon"><i class="fa fa-link"></i></span> Links</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>20 Oct. 2020 to Present</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://humanpangenome.org/">Human Pangenome Reference Consortium</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Associate Member of the <strong>Human Pangenome Reference Consortium</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-link"></i></span> <a href="https://www.nature.com/collections/aebdjihcda">Collection</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>09 July 2021</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="http://web.uniroma2.it/">University of Rome Tor Vergata</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Chair</strong> at the "Deciphering the complexity in neurodegeneration and cancer" workshop</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-file-pdf-o"></i></span> <a href="https://github.com/AndreaGuarracino/andreaguarracino.github.io/blob/main/achievements/PhDWorkshop_DecipheringTheComplexityInNeurodegenAndCancer_Poster_AndreaGuarracino.pdf">Poster</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>07 June 2020 to 30 Aug. 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://summerofcode.withgoogle.com/">Google Summer of Code (GSOC)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Helper</strong> for the "Parallel Graph Traversal for Variation Graphs" project</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-link"></i></span> <a href="https://gsocgraph.blogspot.com/2020/08/final-week-recap-of-my-gsoc-experience.html">Blog</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>20 June 2020</code><br></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.lemonde.fr/blog/binaire/">Lemonde/binaire</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Quote in the newspaper <strong>Le Monde</strong>: SARS-CoV-2 et Covid-19</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="icon"><i class="fa fa-link"></i></span> <a href="https://www.lemonde.fr/blog/binaire/2020/05/06/sars-cov-2-et-covid-19-on-va-jouer-sur-les-mots/">Blog</a></p></td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Last updated 2025-02-15 09:19:33 +0100
</div>
</div>
</body>
</html>
